__tests__/app.test.js

```diff
@@ -32,3 +32,41 @@ describe.only("app", () => {
     });
   });
 });
+
+describe("app", () => {
+  describe("/api/articles", () => {
+    it("200: GET - Responds with an array of article objects. each object should have the following properties: author, title, article_id, topic, created_at, votes, article_img_url, comment_count", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          ({ articles } = response.body);
+          expect(articles).toHaveLength(12);
+          articles.forEach((article) => {
+            expect(typeof article).toBe("object");
+            expect(article).toHaveProperty("comment_count");
+            expect(article).toMatchObject({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+              comment_count: expect.any(String),
+            });
+          });
+        });
+    });
+    it("200: GET - returns the articles sorted by date in descending order", () => {
```

Comment:
nice good use of breaking the testing into separate behaviours, I might have been tempted to take this one step further, and have had the comment count be another test (no need to change but thought for the future)